<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Syndicate</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="tree_view_channels">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
     </font>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tab_view_items">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>481</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
      </font>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QListWidget" name="list_item">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>451</width>
        <height>271</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_message">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="button_save">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="button_mark_read">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>res\read.png</normaloff>
	   res\read.png
	  </iconset>
	  </property>
    </widget>
    <widget class="QToolButton" name="button_mark_all_read">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="icon">
	   <iconset>
       <normaloff>res\readall.png</normaloff>
	   res\readall.png
	  </iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="button_new">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>res\plus.png</normaloff>
	   res\plus.png
	  </iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="button_refresh">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
	 <property name="icon">
      <iconset>
       <normaloff>res\reload.png</normaloff>
	   res\reload.png
	  </iconset>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="text_edit_content">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>390</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string notr="true"/>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="markdown">
     <string notr="true"/>
    </property>
    <property name="html">
     <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
